lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -491,6 +491,10 @@ func dockerContainerCreateParamsToPortlayer(cc types.ContainerCreateConfig, laye
 	config.WorkingDir = new(string)
 	*config.WorkingDir = cc.Config.WorkingDir
 
+	// attach
+	config.Attach = new(bool)
```

Comment:
hmm I think I leave it as it is to keep the consistency with the rest of the code but I agree we should start to switch swag types.

This Attach value gets serialized into guestinfo and then used on the other side (tether)